backend/clubs/permissions.py

```diff
@@ -78,9 +119,8 @@ def has_permission(self, request, view):
                 return False
             if not request.user.is_authenticated:
                 return False
-            membership = Membership.objects.filter(
-                person=request.user, club__code=view.kwargs["club_code"]
-            ).first()
+            obj = Club.objects.get(code=view.kwargs["club_code"])
+            membership = find_membership_helper(request.user, obj)
```

Comment:
Generally, a good goal is to minimize the number of SQL queries that you perform. Each query adds a nonnegligible amount of delay to the loading time of the request. In fact, there are some cases where performing extra work on the SQL side in order to reduce the number of queries can make your program faster overall.

With this change, you go from 1 query for membership checks to at least 3 queries for a membership check. The code fetches this object from the database just to match the existing input format, which is okay but not optimal for performance. It should be possible to reduce this down to 1 query, but that would not be too easy.

The code right now is fine since refactoring this to be slightly more efficient would be nontrivial effort.